lib/elixir/test/elixir/exception_test.exs

```diff
@@ -259,8 +259,8 @@ defmodule ExceptionTest do
       :exit, reason -> reason
     end
 
-    assert Exception.format_exit(reason) ==
-           "exited in: :gen_server.call(:does_not_exist, :hello)\n    ** (EXIT) no process"
+    expected_to_start_with = "exited in: :gen_server.call(:does_not_exist, :hello)\n    ** (EXIT) no process:"
+    assert Exception.format_exit(reason) |> String.starts_with?(expected_to_start_with)
```

Comment:
Since this test cares about indent I like that it's anchored in the beginning, but I don't feel super strongly, if you feel it should be changed.